oopsla2024-benchmarks/README.md

```diff
@@ -0,0 +1,56 @@
+# OOPSLA 2024 benchmarks for Cedar
+
+## Getting started
+
+We recommend you use Docker for benchmarking, as our Dockerfile automatically
+pulls in all the necessary dependencies.
+
+1. Build the Docker image with `docker build -t cedar-benchmarks .` from this
+directory.
```

Comment:
It would be nice to support native runs of the benchmarks, too, not just in docker. Is that hard? I looked in the Dockerfile and I suppose it basically tells you what you have to do if you want to do it manually.

At AWS, at least on a Mac, we are supposed to use `finch`, not `docker`. I tried to build the Docker image with `finch` but it fails. Perhaps the issue is that running on a Mac it cannot issue Linux-specific commands like `apt-get` ? I don't think so because it seems to being working until it gets to dotnet.
```
 => ERROR [ 4/23] RUN apt-get update && apt-get install -y git python3-matplotlib python3  4.7s 
------                                                                                          
 > [ 4/23] RUN apt-get update && apt-get install -y git python3-matplotlib python3-pandas golang-1.21-go dotnet-sdk-6.0 openjdk-17-jdk vim &&     rm -rf /var/lib/apt/lists/*:                  
1.166 Get:1 http://deb.debian.org/debian bookworm-backports InRelease [56.5 kB]                 
1.198 Get:2 http://deb.debian.org/debian bookworm InRelease [151 kB]                            
1.238 Get:3 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]                   
1.266 Get:4 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
1.290 Get:5 http://deb.debian.org/debian bookworm-backports/main arm64 Packages [183 kB]
1.345 Get:6 http://deb.debian.org/debian bookworm/main arm64 Packages [8685 kB]
2.877 Get:7 http://deb.debian.org/debian bookworm-updates/main arm64 Packages [12.5 kB]
2.891 Get:8 http://deb.debian.org/debian-security bookworm-security/main arm64 Packages [144 kB]
3.222 Get:9 https://packages.microsoft.com/debian/12/prod bookworm InRelease [3618 B]
3.281 Get:10 https://packages.microsoft.com/debian/12/prod bookworm/main arm64 Packages [6183 B]
3.318 Get:11 https://packages.microsoft.com/debian/12/prod bookworm/main amd64 Packages [33.4 kB]
3.336 Get:12 https://packages.microsoft.com/debian/12/prod bookworm/main armhf Packages [5602 B]
3.336 Get:13 https://packages.microsoft.com/debian/12/prod bookworm/main all Packages [342 B]
3.668 Fetched 9384 kB in 4s (2615 kB/s)
3.668 Reading package lists...
4.073 Reading package lists...
4.448 Building dependency tree...
4.541 Reading state information...
4.584 E: Unable to locate package dotnet-sdk-6.0
4.584 E: Couldn't find any package by glob 'dotnet-sdk-6.0'
4.584 E: Couldn't find any package by regex 'dotnet-sdk-6.0'
------
Dockerfile:6
--------------------
   5 |         rm packages-microsoft-prod.deb
   6 | >>> RUN apt-get update && apt-get install -y git python3-matplotlib python3-pandas golang-1.21-go dotnet-sdk-6.0 openjdk-17-jdk vim && \
   7 | >>>     rm -rf /var/lib/apt/lists/*
   8 |     ENV PATH="${PATH}:/usr/lib/go-1.21/bin"
--------------------
error: failed to solve: process "/bin/sh -c apt-get update && apt-get install -y git python3-matplotlib python3-pandas golang-1.21-go dotnet-sdk-6.0 openjdk-17-jdk vim &&     rm -rf /var/lib/apt/lists/*" did not complete successfully: exit code: 100
```

Reply:
FYI I was able to build the image on my Mac after commenting out some stuff in the docker image. Sent up a mod to the `Dockerfile`.

Reply:
Don't know what's going on there.  (I develop on Linux, not Mac, but Docker should work equally well either place.)  Thanks for your patch, I believe this is resolved now.